### PR TITLE
benchmark: add description to explain the non-index test result

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

+ benchmark: add description to explain the non-index test result
+ Fix a typo: Region Group --> Raft Group

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5719 https://github.com/pingcap/docs-cn/pull/5689
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/v4.0-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -163,6 +163,10 @@ To avoid TiKV and TiFlash racing for disk and I/O resources, mount the two NVMe
 
 ## Test result
 
+> **Note:**
+>
+> The tables on which SQL statements are executed in this test only has the primary key and does not have secondary index. Therefore, the test result below is not influenced by index.
```
Comment (Joyinqin) [inline benchmark/v4.0-performance-benchmarking-with-tpch.md:168]: > In this test, the tables on which SQL statements are executed only have the primary keys and do not have secondary indexes. Therefore, the test result below is not influenced by the index.

Comment (CharLotteiu) [inline benchmark/v4.0-performance-benchmarking-with-tpch.md:168]:
```suggestion
> The tables on which SQL statements are executed in this test only have primary keys and do not have secondary indexes. Therefore, the test result below is not influenced by indexes.
```

**Review by CharLotteiu — APPROVED** [review]
rest LGTM

**Review by Joyinqin — CHANGES_REQUESTED** [review]

**Review by Joyinqin — APPROVED** [review]
rest LGTM

**Comment by TomShawn** [comment]
@CharLotteiu PTAL again, thanks!

**Pull request merged**